src/utl/include/utl/Logger.h

```diff
@@ -289,9 +289,11 @@ class Logger
 
     if (count == max_message_print) {
       logger_->log(level,
-                   "[{} {}-{:04d}] message limit reached, "
-                   "this message will no longer print"
-                       + std::string(spdlog::details::os::default_eol),
+                   FMT_RUNTIME("[{} {}-{:04d}] message limit ("
+                               + std::to_string(max_message_print)
```

Comment:
Just put {} in the message, pass default_eol as an arg, and skip FMT_RUNTIME.  The is no user provided string here that requires this approach.